src/registerServiceWorker.js

```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
So this file seems nice enough to have, and is defined in `index.js` but then never used?

Reply:
Whoops... I forgot to call it!